resources/assets/coffee/osu_common.coffee

```diff
@@ -29,6 +29,10 @@
     (body.scrollHeight - body.scrollTop) - body.clientHeight
 
 
+  currentUserIsFriendsWith: (user_id) ->
+    currentUser && _.find currentUser.friends, (o) -> o.target_id == user_id
```

Comment:
`currentUser` is always defined, I believe? Empty object for anonymous. Also can pass `target_id: user_id` instead of function.